dali/operators/reader/nvdecoder/nvdecoder.cc

```diff
@@ -442,19 +443,14 @@ void NvDecoder::convert_frame(const MappedFrame& frame, SequenceWrapper& sequenc
                                       input_width,
                                       input_height,
                                       ScaleMethod_Linear);
-  if (dtype_ == DALI_UINT8) {
-    process_frame<uint8>(textures.chroma, textures.luma,
+  TYPE_SWITCH(dtype_, type2id, OutputType, (float, uint8_t), (
```

Comment:
you could have a
```
#define NVDECODER_SUPPORTED_TYPES (float, uint8_t)
```
at the top and use it here and in the other TYPE_SWITCH below

Reply:
done